### ISSUE TITLE
Accessibility Issue: WCAG 2.0 AA Colour Contrast Issues /iitb-kpi.html

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#740]
**Description**
Multiple highlighted text elements with insufficient contrast  

relates to #714 

**Device Information**  
*Device*:  
*Operating System (incl. version)*:  
*Browser (incl. version)*:  
*Additional Details*:

**Element(s) Information**  
*Page*:  https://sara-sabr.github.io/ITStrategy/iitb-kpi.html
*CSS Selector(s) (if possible and applicable)*:  
```p:nth-child(8) > code```
```p:nth-child(12) > code```
```p:nth-child(19) > code```
```ul:nth-child(22) > li:nth-child(1) > code```
```ul:nth-child(22) > li:nth-child(2) > code```
```ul:nth-child(22) > li:nth-child(2) > code```
```li:nth-child(3) > code```
```ul:nth-child(24) > li:nth-child(1) > code```
```ul:nth-child(24) > li:nth-child(2) > code```
```p:nth-child(28) > code```
```p:nth-child(30) > code```
```p:nth-child(33) > code```
```p:nth-child(34) > code```
```p:nth-child(41) > code```
```p:nth-child(44) > code```

*Additional Identifier(s) (if possible and applicable)*:  
*Additional Details*:  

**Screenshot(s) (if possible)**

**Additional information or thoughts**

As a result of scan #687
